### DISCUSSION TITLE
This website cannot be reached octopi.local

### DISCUSSION THREAD

**Discussion by javierzg** [UnchartedBull/OctoDash#3337] [Q&A]
hello.
Good morning, I ask you if any of you have gone through the same thing, and I had this problem all day without being able to connect to Octoprin. (Octopi.local) from my computer.
I don't have internet problems and I worked on other issues all day, but I couldn't log in to Octoprin.
I hope you can guide me to understand what is happening.

![20221207_163344](https://user-images.githubusercontent.com/74110402/206280915-900e8a83-4332-4954-b2c8-e657917fd504.jpg)

**Comment by jneilliii** [comment]
might want to ask in the [OctoPrint Discord](https://discord.octoprint.org/) or [OctoPrint Community Forum](https://community.octoprint.org/). but for octopi.local to work in windows you have to install the bonjour printing service.

**Reply by javierzg** [reply]
Thanks for your answers.
It is strange what you say about Bonjour and I have never installed it and I have been connecting to octopi.local for more than a year, but today it has been impossible.
I have already left my problems in Discord and community
Thank you.
